index.css

```diff
@@ -1,154 +1,154 @@
-* {
-  margin: 0;
-  padding: 0;
-  scroll-behavior: smooth;
-  font-family: Verdana, Geneva, Tahoma, sans-serif;
-  outline: none;
-  border: none;
-  text-decoration: none;
+*{
+    margin: 0;
+    padding: 0;
+    scroll-behavior: smooth;
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+    outline: none; border: none;
+    text-decoration: none;
 }
 
-html {
-  font-size: 60%;
-  scroll-behavior: smooth;
-  scroll-padding-top: 6rem;
-  overflow-x: hidden;
+html{
+    font-size: 60%;
+    scroll-behavior: smooth;
+    scroll-padding-top: 6rem;
+    overflow-x: hidden;
 }
 
-.header {
-  position: sticky;
-  top: 0;
-  display: flex;
-  align-items: center;
-  justify-content: space-between;
-  background-color: white;
+.header{
+    position: sticky;
+    top: 0;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    background-color: white;
 }
 
-.header .header__left .fa-bars {
-  font-size: 2rem;
-  color: #555;
-  border-radius: 0.5rem;
-  padding: 1.5rem 1.5rem;
-  cursor: pointer;
-  border: 1rem  rgba(18, 18, 18, 0.2);
+.header .header__left .fa-bars{
+    font-size: 2rem;
+    color: #555;
+    border-radius: .5rem;
+    padding: 1.5rem 1.5rem;
+    cursor: pointer;
+    border: 1rem  rgba(18, 18, 18, 0.2);
 }
 
-.header #toggler {
-  display: none;
+.header #toggler{
+    display: none;
 }
 
-.header__left {
-  display: flex;
-  flex: 0.1;
+.header__left{
+    display: flex;
+     flex: 0.1;
 }
 
-.header__left .fas .fa-bars {
-  padding: 4px;
+.header__left fa-bars{
```

Comment:
It would be better to select the class instead of the icon name so that the style can be applied.